.travis.yml

```diff
@@ -17,8 +17,10 @@ jobs:
         - cd $TRAVIS_BUILD_DIR/backend
         - npx lockfile-lint --type npm --path package-lock.json -o "https:" -o "file:" --allowed-hosts npm
       install:
-        - npm ci
+        - cd $TRAVIS_BUILD_DIR/shared && npm ci
+        - cd $TRAVIS_BUILD_DIR/backend && npm ci
       script:
+        - npm run build
```

Comment:
Could you help explain why we didn't need the build step previously but have included it now? I guess its to build the shared module, but where was the backend being built previously

Reply:
Yep - this is to build the shared module. Previously there was not need to separate build the backend because `npm test` handles it. However, `ts-jest` [does not](https://github.com/kulshekhar/ts-jest/issues/1648) automatically compile project references.